src/components/studentItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+
+const studentItem = (props) => {
+
+  let {student} = props;
```

Comment:
Here you could do:
```javascript
const studentItem = ({student}) => {
...
}
```

Reply:
Are you sure about this? I just used it as I learned in webapplication1 , but when I tried to change it to what you have written above , I get error.

Reply:
You get the error because you're writing it in a TypeScript file, while the code I wrote is JavaScript.
[Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)